src/com/facebook/buck/util/PathMatchers.java

```diff
@@ -0,0 +1,11 @@
+package com.facebook.buck.util;
```

Comment:
This feels like too broad a location as we try to modularize Buck further. Should probably be at least `com.facebook.buck.jvm`. @styurin probably has the most skin in the game here, though, so I defer to him.

Reply:
I'll wait for his response, then. Tks!

Reply:
But my point here is that this could be a location to store matchers for any kind of file, not only `jvm`s.
If this should be moved to `jvm` package, I'd say best is to separate into two classes, both inside their on language package, `JavaPathMatcher` and `KotlinPathMatcher`

Reply:
Yep, I get that. I don't know what the policy is on "core" pieces of Buck knowing about the various plugins though.